src/images/collections/media.svg

```diff
@@ -0,0 +1,50 @@
+<svg width="256" height="256" viewBox="0 0 256 256" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Just checking — did you run this through https://jakearchibald.github.io/svgomg/ to optimize it?

Reply:
If you haven't done so already, please do